### PR TITLE
Split list to navigation for the left sidebar

### LINKED ISSUE #36915: [BITV] 9.1.3.1a/7.1 - The page only has one "h1" heading. Neither the left-side navigation nor the main content is structured by the headings. (1)
It is suggested to mark the two headings "Personal" and "Administration" as "h2" and to use a visually hidden "h2" heading for the main content, for example, "Logging data." It is recommended to replace the current "h1" heading "Nextcloud's homepage" with a heading that better describes the content of the page, such as "Logging". 

414c728086658dfdb595a873bd04deeb

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-be75bbae42-v7-n1

</details>

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
* Resolves: https://github.com/nextcloud/server/issues/36915

⚠️⚠️⚠️ could be a breaking change (but we are far behind NC25) https://github.com/nextcloud/server/issues/36915#issuecomment-1469810679

## Summary

|  Before   | After                                                        |
|-------------|---------------------------------------------------------------|
| Screenshot from 2023-09-22 10-25-09  | Screenshot from 2023-09-22 10-24-59

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/templates/settings/frame.php [hunk 1]
```diff
@@ -29,58 +29,60 @@
...
-				<?php
+<div id="app-navigation">
+	<?php if (!empty($_['forms']['admin'])): ?>
+		<div id="app-navigation-caption-personal" class="app-navigation-caption-personal"><?php p($l->t('Personal')); ?></div>
+	<?php endif; ?>
+	<nav class="app-navigation-personal" aria-labelledby="app-navigation-caption-personal">
+		<ul tabindex="0">
+			<?php foreach ($_['forms']['personal'] as $form) {
+				if (isset($form['anchor'])) {
+					$anchor = \OC::$server->getURLGenerator()->linkToRoute('settings.PersonalSettings.index', ['section' => $form['anchor']]);
```
[the remark this answers is not in the document]
Comment (JuliaKirschenheuter) [reply apps/settings/templates/settings/frame.php:40]: i haven't changed this part of code

File: apps/settings/templates/settings/frame.php [hunk 2]
```diff
@@ -29,58 +29,60 @@
...
+	</nav>
+
+	<?php if (!empty($_['forms']['admin'])): ?>
+		<div id="app-navigation-caption-administration" class="app-navigation-caption-administration"><?php p($l->t('Administration')); ?></div>
+	<?php endif; ?>
+	<nav class="app-navigation-administration" aria-labelledby="app-navigation-caption-administration">
+		<ul tabindex="0">
+			<?php foreach ($_['forms']['admin'] as $form) {
+				if (isset($form['anchor'])) {
+					$anchor = \OC::$server->getURLGenerator()->linkToRoute('settings.AdminSettings.index', ['section' => $form['anchor']]);
```
[the remark this answers is not in the document]
Comment (JuliaKirschenheuter) [reply apps/settings/templates/settings/frame.php:67]: i haven't changed this part of code

**Review by artonge — APPROVED** [review]

**Comment by JuliaKirschenheuter** [comment]
@artonge do you have any breaking changes in mind?

**Comment by artonge** [comment]
> @artonge do you have any breaking changes in mind?

No

**Review by ShGKme — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by Pytal** [comment]
Drone looks related ;)

File: core/css/apps.scss [hunk 3]
```diff
@@ -138,6 +138,22 @@ kbd {
 	flex-direction: column;
 	flex-grow: 0;
 	flex-shrink: 0;
+}
+.app-navigation-caption,
+.app-navigation-caption {
```
Comment (ShGKme) [inline core/css/apps.scss:143]:
```suggestion
.app-navigation-caption {
```

**Pull request merged**